index.html

```diff
@@ -39,7 +39,124 @@ <h2 class="caption">I’am a Software Developer</h2>
+                <button type="button">See Project</button>                
+            </div>
+            <div class="card-work">
+                <h3>Profesional Art<br> Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>                
+            </div>
+            <div class="card-work">
+                <h3>Profesional Art<br> Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>                
+            </div>
+            <div class="card-work">
+                <h3>Profesional Art<br> Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>                
+            </div>
+            <div class="card-work">
+                <h3>Profesional Art<br> Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>                
+            </div>
+            <div class="card-work">
+                <h3>Profesional Art<br> Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>                
+            </div>
+        </section>
+    </section>
+    <section class="about-me">
+        <div class="me-content">
+            <div class="title"><h2>About Me</h2></div>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button type="button">Get My Resume</button>
```

Comment:
- [ ] From the figma design template you are using, implement the button styling features of Active, Hover and Pressed. Do this for all the buttons.